zipkin-lens/.eslintrc.js

```diff
@@ -12,54 +12,44 @@
  * the License.
  */
 module.exports = {
-  extends: [
-    'airbnb-typescript',
-    'plugin:prettier/recommended',
-    'prettier/@typescript-eslint',
-    'prettier/react',
-  ],
+  extends: ['plugin:prettier/recommended', 'react-app', 'react-app/jest'],
```

Comment:
Remove

* `airbnb-typescript`
* `prettier/@typescript-eslint`
* `prettier/react`

and extend react-scrtips eslint configs.